DomainContent/Hub/RainbowThrower/scripts/flamethrowerClient.js

```diff
@@ -1,643 +1,689 @@
+                        endFireServerCall = "createEndFireHitAvatar";
+                    }
+                    endFireServerArgs = args;
+                } else if (DEBUG_FIRING) {
+                    endFireServerCall = "createEndFireNoHit";
+                    endFireServerArgs = args;
+                }
+                
+                endFireFrameDelay = END_FIRE_FRAME_DELAY;
+            }
+
+            var barrelPosition = this.getBarrelPosition();
+            var firingVolume = this.getFiringVolume();
+            var soundOptions = firingSound.options;
+            soundOptions.position = barrelPosition;
+            soundOptions.volume = firingVolume;
+            firingSound.options = soundOptions;
+
+            var emitAcceleration = this.getRotatedEmitAcceleration();
+            var emitRate = Math.random() * (EMIT_RATE_SPREAD * 2) + BASE_EMIT_RATE;
+            var radiusFinish = this.getRadiusFinish();
+            var particleRadius = this.getParticleRadius();
+            Entities.editEntity(flameEntity, {
+                emitAcceleration: emitAcceleration, 
+                emitRate: emitRate,
+                particleRadius: particleRadius,
+                radiusFinish: radiusFinish
+            });
+            
+            var nozzleLightIntensity = this.getNozzleLightIntensity();
+            Entities.editEntity(nozzleLight, {intensity: nozzleLightIntensity});
+        },
+        
+        stopFiring: function() {
+            if (flameEntity !== null) {
+                Entities.editEntity(flameEntity, {isEmitting: false});
+            }
+            if (nozzleLight !== null) {
+                Entities.deleteEntity(nozzleLight);
+                nozzleLight = null;
+            }
+            if (firingSound !== null) {
+                firingSound.stop();
+                firingSound = null;
+            }
+            isFiring = false;
+            fireKeyPressed = false;
+            endFireServerCall = null;
+            endFireServerArgs = null;
+        },
+        
+        addDesktopOverlay: function() {
+            this.removeDesktopOverlay();
+            
+            if (currentHand === null || !DESKTOP_HOW_TO_OVERLAY) {
+                return;
+            }
+
+            var showOverlay = true;
+            var otherHandDesktopOverlay = this.getOtherHandDesktopOverlay();
+            if (otherHandDesktopOverlay !== null) {
+                desktopHowToOverlay = otherHandDesktopOverlay;
+                showOverlay = false;    
+            }
+            
+            if (showOverlay) {
+                var viewport = Controller.getViewportDimensions();
+                var windowHeight = viewport.y;
+                desktopHowToOverlay = Overlays.addOverlay("image", {
+                    imageURL: DESKTOP_HOW_TO_IMAGE_URL,
+                    x: 0,
+                    y: windowHeight - DESKTOP_HOW_TO_IMAGE_HEIGHT - 24,
```

Comment:
does 24 need to be hard-coded?

Reply:
checking with David...

Reply:
fixed with last commit

Reply:
I don't see the fix in the most recent push